controllers/nodemaintenance_controller.go

```diff
@@ -55,16 +72,379 @@ type NodeMaintenanceReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.9.2/pkg/reconcile
 func (r *NodeMaintenanceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx)
```

Comment:
in master its 
`reqLogger := log.WithFields(log.Fields{"Request.Namespace": request.Namespace, "Request.Name": request.Name})`

and then keeps using that to the rest of the scope.

Reply:
will log.FromContext(ctx) do the same?

Reply:
good question, and yes it does (didn't check where the magic happens though)
`2021-10-01T12:10:30.962962014Z 2021-10-01T12:10:30.962Z	INFO	controller-runtime.manager.controller.nodemaintenance	Reconciling NodeMaintenance	{"reconciler group": "nodemaintenance.kubevirt.io", "reconciler kind": "NodeMaintenance", "name": "nodemaintenance-test-maintenance", "namespace": ""}`